website/docs/r/ess_scaling_configuration.html.markdown

```diff
@@ -106,6 +106,7 @@ The following arguments are supported:
 * `internet_charge_type` - (Optional) Network billing type, Values: PayByBandwidth or PayByTraffic. Default to `PayByBandwidth`.
 * `internet_max_bandwidth_in` - (Optional) Maximum incoming bandwidth from the public network, measured in Mbps (Mega bit per second). The value range is [1,200].
 * `internet_max_bandwidth_out` - (Optional) Maximum outgoing bandwidth from the public network, measured in Mbps (Mega bit per second). The value range for PayByBandwidth is [0,100].
+* `credit_specification` - (Optional) Performance mode of the t5 burstable instance. Valid values: 'Standard', 'Unlimited'.
```

Comment:
The next version is 1.98.0 and if you add a new resource attribute, you should show its available version, like 
```
 `credit_specification` - (Optional, Available in 1.98.0+) 
```

Reply:
Done. 
Let me know if I need to add the same into 'datasource' markdown as well